mypage.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>My Page</title>
+</head>
+
+<body>
+    <div> 
+       <header>
+         <div id="logo">
+                <img src="images/logo.jpg" width="100" height="100" alt="logo" align="left" />
+                <h1>My Paintings</h1>
+         </div>
+        </header>
+    </div>
+
+    <h3><a href="index.html">Home</a> | <a href="bio.html">Tutorials</a> | <a href="schedule.html" target="_blank">Art</a></h3>
+    <hr>
+    <br>
+    <br>
+    <br>
+
+
+<div>
+    <img src="images/pictura1.jpg"  width="400" height="400" align="lef" width="100" alt="" /> 
+
+      <h1>Liniste</h1>
+  
+      <h3>Pictura in ulei, 40cmx30cm</h3>
+      <h3>Price: 50 EUR</h3>
+      <input type="checkbox" name="40/50" value="40/50" />
+		60cmx80cm
+		<input type="checkbox" name="Onions" value="Onions" />
```

Comment:
Onions? :D
De obicei folosim un nume sugestiv. In cazul tau, potrivit ar fi
"dimensiuneMica"

Reply:
sau '60/80"